curator/static/js/study-editor.js

```diff
@@ -850,10 +850,6 @@ function loadSelectedStudy() {
             if (['^ot:messages'] in data.nexml) {
                 data.nexml['^ot:messages'].message = 
                     makeArray(data.nexml['^ot:messages'].message);
-            } else {
```

Comment:
No sense adding this "local" message store now that we're back to keeping all messages inside their parent annotationEvents.